Otsu_Thresholding.py

```diff
@@ -0,0 +1,65 @@
+import matplotlib.pyplot as plt
+from skimage.io import imread
+from skimage.filters import threshold_otsu
+
+# -*- coding: utf-8 -*-
+"""
+Created on Wed May  1 11:08:50 2019
+Global Otsu thresholding
+@author: mostly Luis
+"""
+
```

Comment:
The file should start with the encoding (although this is not necessary anymore in python3), then comes the info on the file (author etc.) and finally the imports right infront of the code